tests/test_web_urldispatcher.py

```diff
@@ -85,26 +65,26 @@ def teardown():
         assert read_ == data
 
 
-async def test_follow_symlink(tmp_dir_path, aiohttp_client) -> None:
+async def test_follow_symlink(tmp_path, aiohttp_client) -> None:
     """
     Tests the access to a symlink, in static folder
     """
     data = 'hello world'
 
-    my_dir_path = os.path.join(tmp_dir_path, 'my_dir')
-    os.mkdir(my_dir_path)
+    my_dir_path = tmp_path / 'my_dir'
+    my_dir_path.mkdir()
 
-    my_file_path = os.path.join(my_dir_path, 'my_file_in_dir')
-    with open(my_file_path, 'w') as fw:
+    my_file_path = my_dir_path / 'my_file_in_dir'
+    with my_file_path.open('w') as fw:
         fw.write(data)
 
-    my_symlink_path = os.path.join(tmp_dir_path, 'my_symlink')
-    os.symlink(my_dir_path, my_symlink_path)
+    my_symlink_path = tmp_path / 'my_symlink'
+    os.symlink(str(my_dir_path), str(my_symlink_path))
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.symlink_to

Reply:
I have not been able to get that to work in 3.5 due to pytest-dev/pytest/issues/5017

Reply:
See https://github.com/aio-libs/aiohttp/pull/3957#discussion_r311324192